TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/StartDate.cshtml.cs

```diff
@@ -43,9 +47,16 @@ public StartDateModel(TrsLinkGenerator linkGenerator, TrsDbContext dbContext, IC
         _clock = clock;
     }
 
-    public void OnGet()
+    public Task OnGetAsync()
     {
         StartDate = JourneyInstance!.State.StartDate;
+        return JourneyInstance!.UpdateStateAsync(state =>
```

Comment:
Shouldn't be doing updates in a Get method .. and initial state setting should be done in `EnsureInitializedAsync`